example/pubsub/README.md

```diff
@@ -0,0 +1,47 @@
+# Dapr PubSub Example with go-sdk
+
+This folder contains two go file that uses this go-SDK to invoke the Dapr PubSub API.
+
+## Helpful
+![](https://i.loli.net/2020/08/23/5MBYgwqCZcXNUf2.jpg)
+
+## Step
+
+### Prepare
+- Dapr installed
+- A local Redis in Docker.
+May be helpful: `docker run d --name dapr-redis -p 6379:6379 redis --requirepass "123456"`
```

Comment:
Given that since Dapr starts with local redis container, this is not necessary.

Reply:
thanks, I've deleted it.